classifier_and_summarizer/summarizer/text_media_matching_preprocessor.py

```diff
@@ -0,0 +1,99 @@
+''' This module pre-processes the Input
+for the text media matcher
+'''
+from numpy import maximum as matrix_maximum
+from sklearn.metrics.pairwise import cosine_similarity
+
```

Comment:
Now we have 3 files for text_media_matching. Would it make sense to create a separate directory? Also, TextMediaMatcher.py could be renamed to reflect that it deals with preference matrice's creation.

Reply:
Thanks for pointing this out - created a directory separately for that,
However i feel that leaving it as TextMediaMatcher would be preferable since its main functionality is to return the matched text and media and not the preference matrix creation - which is just a pre-processing step to apply the matching.  Let me know if you think there's a better way to do this - would be great to hear your thoughts on this!

Reply:
Raised an issue to possibly amend this in a later PR!